composer/workflows/airflow_db_cleanup.py

```diff
@@ -376,7 +363,7 @@ def cleanup_function(**context):
         logging.error(e)
         logging.error(
             str(airflow_db_model) + " is not present in the metadata."
-            "Skipping...")
+                                    "Skipping...")
```

Comment:
Weird formatting, I think "Skipping..." should be aligned with "str" according to guidelines for better readability

Reply:
Done.